lib/utils/webpack.config.js

```diff
@@ -106,13 +106,13 @@ module.exports = (program, directory, stage, webpackPort = 1500, routes = []) =>
           }),
         ]
       default:
-        throw new Error('The state requested ' + stage + " doesn't exist.")
+        throw new Error('The state requested ' + stage + ' doesn\'t exist.')
     }
   }
 
   function resolve () {
     return {
-      extensions: ['', '.js', '.jsx', '.cjsx', '.coffee', '.json', '.less', '.toml', '.yaml'],
```

Comment:
Why did you remove support for cjsx and coffee? Gatsby sites can still be using modules of these types.